charts/jaeger-operator/COMPATIBILITY.md

```diff
@@ -0,0 +1,20 @@
+The following table shows the compatibility of `Jaeger Operator helm chart` with different components, in this particular case we shows Jaeger Operator, Kubernetes and Strimzi operator compatibility. Cert-manager installed or certificate for webhook service in a secret is required in latest versions of helm chart.
```

Comment:
> in latest versions of helm chart

Maybe change this to "in version 2.29.0+ of the helm chart" so that this ages well, explicit versions seem better than "latest".

Reply:
Sure, added